services/core-web/src/components/common/RenderCheckbox.js

```diff
@@ -18,7 +18,7 @@ const propTypes = {
 
 const RenderCheckbox = (props) => (
   <Form.Item validateStatus={props.meta.touched ? props.meta.error && "error" : ""}>
-    <Checkbox id={props.id} checked={props.input.value} {...props.input} disabled={props.disabled}>
+    <Checkbox id={props.id} {...props.input} disabled={props.disabled}>
```

Comment:
Is the `checked `property not valid anymore in the latest antd? Just kind of confused how removing it fixed the issue because in their docs I think they still list it lol.

Reply:
I have some suspicions but I'm not 100% on the _why_.
- checked is still listed on the documentation, and we do have other checkboxes using that attribute that are working just fine
- the value was actually getting changed correctly, it just wasn't rendering it properly (console logging the form values showed that checking it did change the value)
- the props.input also passes a "value" that I think was in conflict with the checked attribute, I think the combination of checked/input.value/input.onChange was just not working.

Reply:
I see, good find!